packages/react/src/lib/mergeThemes.ts

```diff
@@ -223,9 +223,9 @@ const mergeThemes = (...themes: ThemeInput[]): ThemePrepared => {
     // Use the correct renderer for RTL
     acc.renderer = acc.rtl ? felaRtlRenderer : felaRenderer
 
-    acc.fontFaces = mergeFontFaces(...acc.fontFaces, ...next.fontFaces)
+    acc.fontFaces = mergeFontFaces(...(acc.fontFaces || []), ...(next.fontFaces || []))
```

Comment:
actually there is no need to provide default values for accumulated theme (`acc`), as the process is guaranteed to start with initialized fields (and there shouldn't be any field reset back to non-initialized value during the process)

Reply:
```
 FAIL  packages/react/test/specs/components/Dropdown/Dropdown-test.tsx
  ● Test suite failed to run

    TypeError: Invalid attempt to spread non-iterable instance

      227 |     acc.renderer = acc.rtl ? felaRtlRenderer : felaRenderer
      228 | 
    > 229 |     acc.fontFaces = mergeFontFaces(...acc.fontFaces, ...next.fontFaces)
          |                     ^
      230 | 
      231 |     acc.staticStyles = mergeStaticStyles(...acc.staticStyles, ...next.staticStyles)
      232 | 

      at _nonIterableSpread (../../node_modules/@babel/runtime/helpers/nonIterableSpread.js:2:9)
      at _toConsumableArray (../../node_modules/@babel/runtime/helpers/toConsumableArray.js:8:60)
      at src/lib/mergeThemes.ts:229:21
```

Reply:
It throws on `next.*` actually, updated to:
```
acc.fontFaces = mergeFontFaces(...acc.fontFaces, ...(next.fontFaces || []))
acc.staticStyles = mergeStaticStyles(...acc.staticStyles, ...(next.staticStyles || []))
```

Better ideas? 🤔

Reply:
yes - but isn't it that `acc.fontFaces` is always an array, given the value empty theme is initialized with?

```js
const emptyTheme = {
    siteVariables: {},
    componentVariables: {},
    componentStyles: {},
    fontFaces: [],
    staticStyles: [],
    icons: {},
    animations: {},
  } as ThemePrepared
```

Thus, I would expect that fallback value to be necessary for `next` fonts only

Reply:
Got it, have fixed already 👍